src/plugins/cache.js

```diff
@@ -107,6 +127,21 @@ export default class Cache {
 
   async fetchManagers (...managers: Manager[]): Promise<Plugin[]> {
     let plugins = []
+    if (this.cache.node_version !== process.version) {
+      let lock = new Lock(this.out)
+      let handled = true
+
+      let downgrade = await lock.upgrade()
+      for (let manager of managers) {
+        handled = await manager.handleNodeVersionChange() && handled
```

Comment:
alternatively maybe we should store the node version differently for user plugins and linked plugins?